index.html

```diff
@@ -610,57 +598,79 @@
-            <code>property="schema:author"</code> or <code>property="schema:contributor"</code>,
-            <code>typeof="schema:Person"</code>, and a <code>resource</code> attibute that is a URL
-            identifying the person (typically their home page, or possibly email address — it could
-            also be a URL to their ORCID).
+            <code>property="schema:author"</code> or <code>property="schema:contributor"</code> and a
+            <code>typeof="sa:ContributorRole"</code>.
           </p>
           <p>
-            Inside of that <code>li</code> arbitrary properties of <code>schema:Person</code> can be
-            specified, but providing at least <code>schema:givenName</code> and
-            <code>schema:familyName</code>, as well as ideally <code>schema:email</code> is very
-            much recommended.
+            A <code>sa:ContributorRole</code> type (following the
+            semantic of
+            schema.org <a href="http://schema.org/Role">Role</a>) is
+            used so that affiliations or contact informations (email
+            address, etc.) relevant to this specific scholarly article
+            (and this specific scholarly article only) can be
+            specified. This is important as authors may have different
+            affiliations and contact points at the time they are
+            publishing a scholarly article but may want to specify
+            only a subset of those. Readers not familiar with the
+            semantic of schema.org Role can consult
+            the <a href="http://blog.schema.org/2014/06/introducing-role.html">introductory
+            blog post</a>.
           </p>
           <p>
-            If there is an affiliations <code>ol</code> and a given author is affiliated, there must
-            be an <code>a</code> element with its <code>href</code> pointing to that affiliation, a
-            <code>resource</code> matching the URL identifying the affiliation, and
-            <code>property="schema:affiliation"</code>. The content of that <code>a</code> element
-            must be a string that matches the one that will be generated by CSS to label the
-            affiliation; Latin lowercase being recommended. (This is a hack, but we can only do so
-            much within the limits of CSS — better <code>counter</code> would be needed.)
+            Inside of that <code>li</code>, arbitrary properties
+            of <code>schema:Person</code>, filling
+            the <code>schema:author</code> property of
+            the <code>sa:ContributorRole</code> can be specified, but
+            providing at least <code>schema:givenName</code> and
+            <code>schema:familyName</code>. It is recommended that
+            these properties are wrapped into an hyperlink
+            identifying the person with a URL to their home page,
+            their <a href="http://orcid.org/">ORCID</a>, or an email
+            address.
```

Comment:
I think that this is conflating uses in a manner that doesn't work to model real-world scenarios. What the "corresponding author" does varies from field to field and publishing process to publishing process. We need to accommodate more than one. I have seen it being used for:

* The person that the *editor* contacts if there is a problem during pre-publish processing.
* The person that third parties should contact with inquiries.
* The primary author (in fields where the authors list is alphabetised).

Additionally, pre-prints commonly do not specify a corresponding author, but would have contact information for people nevertheless (it seems that in CompSci you always include *everyone's* email addresses, because, well, computers and stuff).

Conflating email and corresponding author is semantic overloading; IMHO a big red flag!